packages/pages/src/upgrade/pagesUpdater.test.ts

```diff
@@ -0,0 +1,28 @@
+import { describe, it, expect } from "vitest";
+import { removeFetchImport } from "./pagesUpdater.js";
+import { Project } from "ts-morph";
+import path from "path";
+import fs from "fs";
+
+describe("test pages updater steps", () => {
+  it("remove fetch imports", () => {
+    const testFilePath = "src/upgrade/test/src/templates/location.tsx";
```

Comment:
Can you instead have some consts defined so you don't need ts-morph in the test and then tests all of those different fetch import cases I mentioned before?

Reply:
The consts would have to be defined in the fetch function. Do we want to use constants rather than read the ts-config?